app/subscriber/src/store/slices/admin/interfaces/IAdminState.ts

```diff
@@ -31,7 +31,7 @@ export interface IAdminState {
   topics: ITopicModel[];
   rules: ITopicScoreRuleModel[];
   tags: ITagModel[];
-  alerts: IAlertModel[];
+  systemMessages: IAlertModel[];
```

Comment:
Change IAlertModel to `ISystemMessageModel`

Reply:
Also delete the `IAlertModel` it is not needed anymore

Reply:
Updated, thanks